src/pages/views/components/charts/historicalCostChart/historicalCostChart.test.tsx

```diff
@@ -38,17 +35,14 @@ const props: HistoricalCostChartProps = {
 };
 
 test('reports are formatted to datums', () => {
-  const view = shallow(<HistoricalCostChart {...props} />);
-  const charts = view.find(ChartArea);
+  const view = render(<HistoricalCostChart {...props} />);
+  const charts = screen.getAllByText(/cost/i);
   expect(charts.length).toBe(4);
-  expect(charts.at(0).prop('data')).toMatchSnapshot('current month cost data');
-  expect(charts.at(1).prop('data')).toMatchSnapshot('current month infrastructure cost data');
-  expect(charts.at(2).prop('data')).toMatchSnapshot('previous month cost data');
-  expect(charts.at(3).prop('data')).toMatchSnapshot('previous month infrastructure cost data');
```

Comment:
the previous test was only testing the props getting sent to the component, rather than what was getting output onto the DOM. I had to do a lot of outputting the screen to see what was actually happening, and often fixing it as nothing at all was displayed. in this case after getting it to work, it's outputting 4 charts with cost in their header so i went with that..

the previous snaps were showing like this:

> exports[`reports are formatted to datums: current month cost data 1`] = `Array []`;
> exports[`reports are formatted to datums: current month infrastructure cost data 1`] = `Array []`;
> exports[`reports are formatted to datums: previous month cost data 1`] = `Array []`;
> exports[`reports are formatted to datums: previous month infrastructure cost data 1`] = `Array []`;

Reply:
I'll revisit this. Was hard testing when the previous tests didn't have the data so I had to try figuring out what was supposed to happen.

Reply:
Ah, it's the line below this comment that is comparing the snapshot which contains the data for the charts.

The getAllByText() is just in addition to this, not necessarily needed, but perhaps a little of a safeguard against someone just doing `test -u` to update snapshots without checking first.

I suppose the test isn't properly named anymore as we are looking at the output (charts themselves in the dom) rather than the data creating them. I can update that.